src/test/ui/async-await/conditional-and-guarnateed-initialization.rs

```diff
@@ -0,0 +1,18 @@
+// check-pass
```

Comment:
There's a typo in the filename ("guarnateed").

Reply:
dang will fix now

Reply:
Updated